src/constants/currency.js

```diff
@@ -72,6 +72,29 @@ export const SYNTHS = [
 	'iDEFI',
 ];
 
+const CRYPTO_SYNTHS = [
+	'sBTC',
```

Comment:
would be great if you could use the SYNTH_MAP for these values.. also the name should probably be something like `CRYPTO_SYNTHS_BY_MC` ? to indicate that this list is not just a list, but its ranked by MC.